codemagic.yaml

```diff
@@ -0,0 +1,28 @@
+workflows:
+  release:
+    environment:
+      vars:
+        GH_USER: madispp
+        GH_PAT: Encrypted(Z0FBQUFBQmQxcFUxdDNKMVNkOGdENlZrdVdKUkxwaTdlWVNPQU56LTczVjFoZk1xalVTVlh1YzhLRFotaGFIemF4NkdYN3BicHdRcmxJcmZhbG5rb1lTelFsUXZCY0ZUN0U3ZW4zTEoyN0FqX2wxUUZKdV9LVTJPLVF5aXpzekMzOWUyTkNIX3Vpam8=)
+        REPOSITORY: codemagic-ci-cd/cli-tools
+    name: Generate binary distribution wheel
+    scripts:
+      - git tag -a v$(python3 setup.py --version) -m "Release v$(python3 setup.py --version)"
+      - python3 setup.py bdist_wheel
+      - git push "https://$GH_USER:$GH_PAT@github.com/$REPOSITORY" -tags
+      - echo "Initial release\n" > release_notes.txt
+      - |
+        # draft a release
+        set -e
+
+        TAG_NAME=v$(python3 setup.py --version)
+        RELEASE_NOTES=$(cat release_notes.txt)
+        PAYLOAD="{\"tag_name\":\"$TAG_NAME\",\"name\":\"$TAG_NAME\",\"body\":\"$RELEASE_NOTES\",\"draft\":true,\"prerelease\":true}"
+        RELEASE=$(curl -u "$GH_USER":"$GH_PAT" --silent --fail -H 'Content-Type: application/json' -X POST --data "$PAYLOAD" "https://api.github.com/repos/${REPOSITORY}/releases")
+        EDIT_URL=$(echo $RELEASE | python -c 'import sys, json; print json.load(sys.stdin).get("html_url").replace("/tag/", "/edit/")')
```

Comment:
Absolutely. I would even prefer it, but AFAICT `jq` isn't present on our builders. Should we add it?
Otherwise I can fix it in a separate change after we get jq.

Reply:
Indeed, `jq` is not installed at the moment :( I think we should include it with next update, and leave this as is for now.